runtime/src/accounts.rs

```diff
@@ -345,8 +345,8 @@ impl Accounts {
         })
     }
 
-    pub fn verify_hash_internal_state(&self, slot: Slot, ancestors: &HashMap<Slot, usize>) -> bool {
-        self.accounts_db.verify_hash_internal_state(slot, ancestors)
+    pub fn verify_bank_hash(&self, slot: Slot, ancestors: &HashMap<Slot, usize>) -> bool {
+        self.accounts_db.verify_bank_hash(slot, ancestors).is_ok()
```

Comment:
At the very layer boundary of "Bank<=>AccountsDB", we discard detailed error and turn the result into a bool.